### ISSUE TITLE
Esc menu

### ISSUE THREAD

**Issue by tobyplowy** [DreamBlocks/DreamBlocks#32]
If you press esc now it quits the game
A menu shuld popup and have a copple options
Like 

Resume game (closes the esc menu)
Options (opens options for sound Graphics and so on)
quit game (go's to desktop)
quit to menu (title screen)

**Comment by tobyplowy** [comment]
@LordStephen77 I think this should be done early in development 
I whant this soon

**Comment by AKryukov92** [comment]
When you press esc you go to main screen. Once there, you can use "load" button to resume game.
Though, It is possible to let the game to "resume". To do this, we need two sprites for button "resume". One for "up" state" and one for "down" state.

**Comment by LordStephen77** [comment]
@AKryukov92 Exact! you are right. In practice, the game is recharged.. In practice, the game is recharged.

I propose to create more beautiful aesthetic buttons so that everyone can change them all. If no one is willing, I will do it.

Thank you.

**Comment by tobyplowy** [comment]
Why not just add two variants of the button 
Pressed in and not pressed in
And just overlay the text (font) on top of the button
Now we are just make a 100 buttens but we can do with 2

**Comment by tobyplowy** [comment]
> Pressed in and not pressed in

Maybe also a highlight butten

**Comment by AKryukov92** [comment]
There was a bug with this menu: when you click anywhere on the screen in main menu, the game closes. It should be fixed in https://github.com/DreamBlocks/DreamBlocks/commit/0e2f20bbbeb126bb60c69af6f267b1eb5c53cfb9
